common.yaml

```diff
@@ -242,6 +242,8 @@ packages:
  - kernel-modules-extra
  # Audit
  - audit
+ # Bootloader updater
+ - bootupd
```

Comment:
We already include the same `bootupd.yaml` manifest in this file. We just need to move it out from the `arch-include` and into the `include` section.

BTW, to confirm the no-op s390x support will also correctly no-op on the `bootupctl backend generate-update-metadata` step?